src/logger.ts

```diff
@@ -1,63 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license.
-
-import * as fs from "fs";
-import * as vscode from "vscode";
-import TelemetryReporter from "vscode-extension-telemetry";

```

Comment:
The dependency of `vscode-extension-telemetry` can also be removed in package.json

Reply:
done.